### ISSUE TITLE
подписка на котировки и сделки

### ISSUE THREAD

**Issue by nableru** [FinamWeb/trade-api-docs#9]
Правильно ли я понял после чтения документации, что API не предназначен для использования с WebSockets, поэтому нет возможности получать в реальном времени ни стакан, ни сделки?

**Comment by DBoyara** [comment]
нет пока что у них такого функционала

**Comment by Dronablo** [comment]
Судя по последним коммитам - появилась возможность получать стакан по подписке через gRPC.

- Так ли это?
- Т.к. поток положить в HTTP - не тривиально, не планируется ли в REST чего-то вроде `https://trade-api.finam.ru/api/v1/marker_data/get_current_orderbook?code=SU29008RMFS8&board=TQOB&market=Stock&max_depth=10`?

**Comment by Vkhalchenya** [comment]
@Dronablo 
Да, мы добавили возможность получать стакан по подписке через gRPC.
Делать стакан в REST не планировали, но в следующем релизе запланировано добавить получение исторических рыночных данных.

**Comment by Dronablo** [comment]
Пока немного не хватает полноты доки:
* Вот я подписался на стакан.
* Мне пришел первый OrderBookEvent.
* Гарантируется ли что в нем все заявки из стакана без ограничения глубины на момент "сразу после подписки"?

**Comment by Vkhalchenya** [comment]
Метод отдает стакан 20 на 20 - это весь объем стакана, который предоставляет Transaq как терминал.
